doc/sphinx/source/shared/index.rst

```diff
@@ -0,0 +1,24 @@
+.. shared/index: 
+
+Subgraphs 
+=========
+
+As the toolbox for Deep Learning (DL) developers becomes more powerful and 
+better able to target workloads with both local and remote data, it becomes 
+increasingly important to be able to work in collaborative spaces like 
```

Comment:
I don't really understand this sentence... not sure what you mean by "collaborative spaces".

Reply:
Another remnant phrase from the blog we never published, I think it can be cut here too